### PR TITLE
Maintainer list

### PR DESCRIPTION
**Pull request by kannon92**
Partially: #1591 

Add a MAINTAINER.md to your repo.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| Kevin Hannon         | [kannon92](https://github.com/kannon92)                 | G-Research  |
+| Clifton Houck        | [ClifHouck](https://github.com/ClifHouck)               | G-Research  |
+| Chris Martin         | [d80tb7](https://github.com/d80tb7)                     | G-Research  |
+| James Murkin         | [JamesMurkin](https://github.com/JamesMurkin)           | G-Research  |
+| Dejan Zele Pejchev   | [dejanzele](https://github.com/dejanzele)               | G-Research  |
+| Jamie Poole          | [jimbobby5](https://github.com/jimbobby5)               | G-Research  |
+| Alex Scammon         | [stackedsax](https://github.com/stackedsax)             | G-Research  |
+| Rich Scott           | [richscott](https://github.com/richscott)               | G-Research  |
+| Albin Severinson     | [severinson](https://github.com/severinson)             | G-Research  |
+| Daniel Rastelli      | [theAntiYeti](https://github.com/theAntiYeti)           | G-Research  |
```
Comment (richscott) [inline MAINTAINERS.md:19]: It looks like this is sorted alphabetically by last name - should Daniel Rastelli be after Jamie Poole?

Comment (kannon92) [reply MAINTAINERS.md:19]: Doh

**Review by richscott — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (suprjinx) [reply MAINTAINERS.md]: This is why my kids got their mother's last name (starts with P, middle of such lists) instead of mine (Wilson, always last) :-)

**Comment by dave-gantenbein** [comment]
@suprjinx don't worry, you're still the first one I yell at!

**Review by suprjinx — APPROVED** [review]

**Pull request merged**